signer/WebCryptoSigner.js

```diff
@@ -62,6 +62,24 @@ class WebCryptoSigner {
   /**
    * @param {Uint8Array} sig
    * @return {Uint8Array}
+   *
+   * This function refer to the SDK of token.io (https://developer.token.io)
```

Comment:
I'm not sure but is there more specific URL rather than https://developer.token.io ?
I can't find the sample code you referred.

Also, is it updated based on the sample code or just taken (copied) from it ?
If it's just copied, the following sounds more clear.
```suggestion
   * This function is taken from the SDK of token.io (https://developer.token.io)
```

Reply:
@feeblefakie 
This source code is in https://github.com/tokenio/sdk-js
Sorry, I just realize that token.io doesn't declare the LICENSE.
Therefore we shouldn't copy their code to use.
I provide my own implementation of the function `_P1363ToDer`.
Please take a look.
Sorry for the lack of confirmation.

Reply:
It is ISC license. So we can copy the code.

Reply:
Ok. Revised.
I also made more comments on that.
Please kindly review.

Reply:
Also added copyright declaration.